ads/google/a4a/performance.js

```diff
@@ -250,4 +250,12 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
     this.element_.parentNode.insertBefore(pingElement, this.element_);
     dev().info('PING', url);
   }
+
+  /**
+   * Returns the initialization time of this reporter.
+   * @return {number} The initialization time in ms.
+   */
+  getInitTime() {
+    return this.initTime_;
+  }
```

Comment:
Is there a special reason we have another `getDeltaTime()` that call `getDeltaTime_()`, instead of just make `getDeltaTime_` a public method?
I am good with current approach though, if we will be removing calling of it later.